tools/codesign_mac_executable.py

```diff
@@ -0,0 +1,232 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+# Please refer to https://medium.com/@parthnaik92/codesign-and-notarize-unreal-engine-mac-builds-for-distribution-outside-of-the-mac-store-d2f6e444f3a7
+# for pre-requisites and setup your system before trying to run this file.
+
+import argparse
+import os
+import shutil
+import subprocess
+import time
+
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--macos_executable", required=True)
```

Comment:
- rename to sign_executable.py
- rename to args.executable
- assert sys.platform == "darwin" at the beginning of the script

Reply:
Done

Reply:
call the folder sign_executable

Reply:
Done